src/main/java/teammates/ui/controller/InstructorSearchPageAction.java

```diff
@@ -113,11 +113,12 @@ private int filterFeedbackResponseCommentResults(
                 InstructorAttributes instructor = this.getInstructorForCourseId(response.courseId, instructors);
                 
                 boolean isVisibleResponse = true;
-                boolean isNotAllowedForInstructor = instructor == null
-                                                    || !(instructor.isAllowedForPrivilege(response.giverSection, response.feedbackSessionName,
-                                                                                           Const.ParamsNames.INSTRUCTOR_PERMISSION_VIEW_SESSION_IN_SECTIONS))
-                                                    || !(instructor.isAllowedForPrivilege(response.recipientSection, response.feedbackSessionName,
-                                                                                           Const.ParamsNames.INSTRUCTOR_PERMISSION_VIEW_SESSION_IN_SECTIONS));
+                boolean isNotAllowedForInstructor =
+                        instructor == null
+                        || !(instructor.isAllowedForPrivilege(response.giverSection, response.feedbackSessionName,
+                                                              Const.ParamsNames.INSTRUCTOR_PERMISSION_VIEW_SESSION_IN_SECTIONS))
+                        || !(instructor.isAllowedForPrivilege(response.recipientSection, response.feedbackSessionName,
+                                                              Const.ParamsNames.INSTRUCTOR_PERMISSION_VIEW_SESSION_IN_SECTIONS));
```

Comment:
@wkurniawan07 I'm refering to something like this (the right hand side can be considered the next level):
~~~
boolean isNotAllowedForInstructor =
            instructor == null
            || !(instructor.isAllowedForPrivilege(response.giverSection, response.feedbackSessionName,
                                                  Const.ParamsNames.INSTRUCTOR_PERMISSION_VIEW_SESSION_IN_SECTIONS))
            || !(instructor.isAllowedForPrivilege(response.recipientSection, response.feedbackSessionName,
                                                  Const.ParamsNames.INSTRUCTOR_PERMISSION_VIEW_SESSION_IN_SECTIONS));
~~~